src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/help.py

```diff
@@ -11,19 +11,141 @@
             type: group
             short-summary: Commands to manage Azure Monitor service.
             """
-# MANAGEMENT COMMANDS HELPS
-helps['monitor alert-rules'] = """
-            type: group
-            short-summary: Commands to manage alerts assigned to Azure resources.
-            """
-helps['monitor alert-rules update'] = """
-            type: command
-            short-summary: Updates an alert rule.
-            """
-helps['monitor alert-rule-incidents'] = """
-            type: group
-            short-summary: Commands to manage alert rule incidents.
-            """
+
+# region Alerts
+
+helps['monitor alert'] = """
+    type: group
+    short-summary: Commands to manage metric-based alert rules.
+    """
+
+helps['monitor alert create'] = """
+    type: command
+    short-summary: Create a metric-based alert rule.
+    parameters:
+        - name: --action -a
+          short-summary: Add an action to fire when the alert is triggered.
+          long-summary: |
+            Usage:   --add-action TYPE KEY [ARG ...]
```

Comment:
May be I am missing something here but could you please help me understand how parameter name is `action` and usage is via `add-action` and parameter in the custom method is named as `actions` and that is different from 
https://github.com/Azure/azure-cli/pull/3373/files#diff-64d03a5146944eb9ee80660a3b183ae0R72 in `update` command